test/disable-user-indexes/user-indexes-disabled.td

```diff
@@ -125,3 +133,104 @@ on_name   key_name        seq_in_index  column_name  expression  nullable enable
 -----------------------------------------------------------------------------
 t         t_primary_idx   1             a            <null>      true     false
 t         t_secondary_idx 1             <null>       "a + a"     true     false
+
+# 🔬 Enabling indexes
+
+# 🔬🔬 Views + Sources
+
+> ALTER INDEX logging_derived_mat_primary_idx SET ENABLED;
+
+> SELECT * FROM logging_derived_mat
+1
+
+> SHOW INDEXES FROM logging_derived_mat;
+on_name             key_name                        seq_in_index  column_name expression  nullable enabled
+----------------------------------------------------------------------------------------------------------
+logging_derived_mat logging_derived_mat_primary_idx 1             count       <null>      false    true
+
+# Alter index is functionally idempotent
+> ALTER INDEX logging_derived_mat_primary_idx SET ENABLED;
+
+> SELECT * FROM logging_derived_mat
+1
+
+# 🔬🔬🔬 Materialized views
+
+> ALTER INDEX mat_view_primary_idx  SET ENABLED;
+
+> SHOW INDEXES FROM mat_view;
+on_name   key_name              seq_in_index  column_name  expression  nullable enabled
+---------------------------------------------------------------------------------------
+mat_view  mat_view_primary_idx  1             sum          <null>      true     true
+
+> SHOW MATERIALIZED VIEWS
+logging_derived_mat
+mat_view
+
+> SELECT * FROM mat_view
+1
+
+# 🔬🔬🔬 Materialized sources
+
+> ALTER INDEX mat_data_primary_idx  SET ENABLED;
+
+> SHOW INDEXES FROM mat_data;
+on_name   key_name              seq_in_index  column_name  expression  nullable enabled
+---------------------------------------------------------------------------------------
+mat_data  mat_data_primary_idx  1             a            <null>      false    true
+
+> SHOW MATERIALIZED SOURCES
+mat_data
+
+> SELECT * FROM mat_data
+1
+
+# 🔬🔬🔬 Non-materialized views
+
+# Still cannot select from non-materialized view because some of its indexes are disabled
+
+! SELECT * FROM join_view
+Unable to automatically determine a timestamp for your query; this can happen if your query depends on non-materialized sources
```

Comment:
We should improve this error message to detect if any of the unmaterialized indexes were disabled by user request, and inform the user of the names of the indexes that must be enabled to run this. Not needed in this PR, or maybe ever, just a nice to have.